src/main/java/uk/gov/hmcts/reform/sscs/helper/mapping/HearingsAdjustmentMapping.java

```diff
@@ -0,0 +1,59 @@
+package uk.gov.hmcts.reform.sscs.helper.mapping;
+
+import uk.gov.hmcts.reform.sscs.ccd.domain.HearingOptions;
+import uk.gov.hmcts.reform.sscs.exception.InvalidMappingException;
+import uk.gov.hmcts.reform.sscs.model.hmc.reference.Adjustment;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+import static java.util.Objects.isNull;
+import static org.apache.commons.lang3.ObjectUtils.isEmpty;
+import static uk.gov.hmcts.reform.sscs.model.hmc.reference.Adjustment.HEARING_LOOP;
+import static uk.gov.hmcts.reform.sscs.model.hmc.reference.Adjustment.SIGN_LANGUAGE_INTERPRETER;
+import static uk.gov.hmcts.reform.sscs.model.hmc.reference.Adjustment.STEP_FREE_WHEELCHAIR_ACCESS;
+
+public final class HearingsAdjustmentMapping {
+
+    public static final String CCD_SIGN_LANGUAGE_INTERPRETER = "signLanguageInterpreter";
+    public static final String CCD_HEARING_LOOP = "hearingLoop";
+    public static final String CCD_DISABLED_ACCESS = "disabledAccess";
+
+    private HearingsAdjustmentMapping() {
+
+    }
+
+    public static List<Adjustment> getIndividualsAdjustments(HearingOptions hearingOptions) throws InvalidMappingException {
+        if (isNull(hearingOptions)) {
+            return Collections.emptyList();
+        }
+        return getAdjustments(hearingOptions.getArrangements());
+    }
+
+    public static List<Adjustment> getAdjustments(List<String> ccdAdjustments) throws InvalidMappingException {
+        List<Adjustment> adjustments = new ArrayList<>();
+        if (isEmpty(ccdAdjustments)) {
```

Comment:
(Minor) could move this to be before the list initialisation

Reply:
Still need to return an empty list and initialise an ArrayList, this just does both at once.